app/views/schemes/check_answers.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <%= form_for(@scheme, as: :scheme, method: :patch) do |f| %>
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-three-quarters-from-desktop">
+    <div class="govuk-grid-column-full">
```

Comment:
We can remove this completely as we are not overriding anything that comes from layouts!